modules/insights-operator-enable-ofuscation.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * support/insights-operator/disabling-insights-operator.adoc
+
+
+
+[id="insights-operator-enable-ofuscation_{context}"]
+= Enabling Insights Operator data obfuscation
+
+
+[role="_abstract"]
```

Comment:
We need to remove this "abstract" definition for the time being, as we are not yet on Pantheon 2.

Reply:
removed